app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: int, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+
+    for person in people:
+        current_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
+            current_person.wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"]:
+            current_person.husband = Person.people[person["husband"]]
```

Comment:
The code assumes that the names for 'wife' and 'husband' are always present in the `Person.people` dictionary. Consider adding error handling to manage cases where a name might not be found.